common/containers/Tabs/WalletDecrypt/index.jsx

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import translate from 'translations';
+import KeystoreDecrypt from './KeystoreDecrypt';
+import PrivateKeyDecrypt from './PrivateKeyDecrypt';
+import MnemonicDecrypt from './MnemonicDecrypt';
+import LedgerNanoSDecrypt from './LedgerNanoSDecrypt';
+import TrezorDecrypt from './TrezorDecrypt';
+import ViewOnlyDecrypt from './ViewOnlyDecrypt';
+
+const decryptionChoices = [
```

Comment:
Absolutely agree @dternyak I'll try finding a nice home for it, let me know what you think (and also, be nitpicky please!! I'd like a codebase we're all happy with 😄  )